### PR TITLE
add a wakeup interval to zw074

### PR DESCRIPTION
**Pull request by caseda**
added a wake up interval (21600, every 6 hours) since the default (0) is not in default range of the Wake-Up CC.
Though every battery device should be capable of being on 0 according to the specs, having it default > 0 is better, for users being able to save parameters without having to wake up the device manually every time.

additional: remove GET's, device is asleep 99% of the time.

### CODE REVIEW

File: drivers/ZW074/driver.js [hunk 1]
```diff
@@ -9,71 +9,51 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
 	capabilities: {
 		measure_battery: {
 			command_class: 'COMMAND_CLASS_BATTERY',
-			command_get: 'BATTERY_GET',
```
Comment (RobinBol) [inline drivers/ZW074/driver.js]: Why did you remove the battery get? Now Homey won't get an update the first time the device wakes up after Homey has for example been rebooted, or does the device auto report this every (or more frequent) wake up?

File: drivers/ZW074/driver.js [hunk 2]
```diff
@@ -9,71 +9,51 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
...
-			}
+			},
 		},
 		alarm_motion: {
 			command_class: 'COMMAND_CLASS_SENSOR_BINARY',
-			command_get: 'SENSOR_BINARY_GET',
 			command_report: 'SENSOR_BINARY_REPORT',
-			command_report_parser: report => report['Sensor Value'] === 'detected an event'
+			command_report_parser: report => {
+				if (report && report.hasOwnProperty('Sensor Value')) return ['Sensor Value'] === 'detected an event';
```
Comment (RobinBol) [inline drivers/ZW074/driver.js]: missing `report` at `return ['Sensor Value'] ...`

File: drivers/ZW074/driver.js [hunk 3]
```diff
@@ -9,71 +9,51 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
...
+				return null;
+			},
 		},
 		measure_temperature: {
 			command_class: 'COMMAND_CLASS_SENSOR_MULTILEVEL',
-			command_get: 'SENSOR_MULTILEVEL_GET',
```
Comment (RobinBol) [inline drivers/ZW074/driver.js]: Same as with battery, why remove get?

File: drivers/ZW074/driver.js [hunk 4]
```diff
@@ -9,71 +9,51 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
...
+				if (report && report.hasOwnProperty('Sensor Type') && report.hasOwnProperty('Sensor Value (Parsed)')) {
 					if (report['Sensor Type'] === 'Temperature (version 1)') return report['Sensor Value (Parsed)'];
 				}
 				return null;
-			}
+			},
 		},
 		measure_luminance: {
 			command_class: 'COMMAND_CLASS_SENSOR_MULTILEVEL',
-			command_get: 'SENSOR_MULTILEVEL_GET',
```
Comment (RobinBol) [inline drivers/ZW074/driver.js]: Same as with battery, why remove get?

File: drivers/ZW074/driver.js [hunk 5]
```diff
@@ -9,71 +9,51 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
...
+				if (report && report.hasOwnProperty('Sensor Type') && report.hasOwnProperty('Sensor Value (Parsed)')) {
 					if (report['Sensor Type'] === 'Luminance (version 1)') return report['Sensor Value (Parsed)'];
 				}
 				return null;
-			}
+			},
 		},
 		measure_humidity: {
 			command_class: 'COMMAND_CLASS_SENSOR_MULTILEVEL',
-			command_get: 'SENSOR_MULTILEVEL_GET',
```
Comment (RobinBol) [inline drivers/ZW074/driver.js]: Same as with battery, why remove get?

**Comment by caseda** [comment]
I have never seen the Get's being queued until the device is awake?, that would make my queue for z-wave devices enormous, has that changed in one of the latest updates?
only seen it happen with parameters.

but the device does indeed send all data on user definable interval and thresholds (param: 111, on default set by homey on 300 seconds, with a set-able threshold params: 41, 42, 43, 44) that is why the default wake up interval of the device = 0

**Comment by RobinBol** [comment]
When a driver is initiated, Homey will GET all the capabilities. So in the case of z-wave devices the GET commands will be send, if the device is not awake at that moment they are queued until the device wakes up. I think it would be good to leave the GETs in, if people start to play with the intervals we still would like Homey to request the data of the device at least when it boots.

File: drivers/ZW074/driver.js (hunk 2 above)
Comment (caseda) [inline drivers/ZW074/driver.js]: whoops forgot that broken something 😊
